src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -214,6 +214,46 @@ protected function sort_events( $events ) {
 
 	}
 
+	/**
+	 * Creates a HTML link and "fast forward" message to append to the "no events found" message.
+	 *
+	 * @since TBD
+	 *
+	 * @param bool  $canonical         Whether to return the canonical (pretty) version of the URL or not.
+	 * @param array $passthru_vars     An optional array of query variables that should pass thru the method untouched
+	 *                                 in key and value.
+	 * @return string                  The html link and message.
+	 */
+	public function get_fast_forward_link( $canonical = false, array $passthru_vars = [] ) {
+		$date      = $this->context->get( 'event_date', $this->context->get( 'today' ) );
+		$cache_key = __METHOD__ . '_' . md5( wp_json_encode( array_merge( [ $date, $canonical ], $passthru_vars ) ) );
+
+		if ( isset( $this->cached_urls[ $cache_key ] ) ) {
+			return $this->cached_urls[ $cache_key ];
+		}
+
+		$next_event = (array) tribe_events()->where( 'starts_after', $date )->per_page( 1 )->fields( 'ids' )->first();
+		$next_event = tribe_get_event( array_shift( $next_event ) );
+
+		if ( empty( $next_event ) ) {
+			return false;
+		}
+
+		$url_date   = Dates::build_date_object( $next_event->start_date );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 3 spaces

<sub>hash: b671f573d104917e3b567a7c9c138370</sub>